improve: feedback on grant roles (#1853)
infrahq/infra: docs/guides/granting-access.md

## Roles

> Infra allows granting different levels of access via **roles**, such as `view`, `connector` or `admin`. Each connector has different roles that can be used:

Reviewer: ```suggestion
Infra allows granting different levels of access via the **roles** `view` or `admin`. Each connector has different roles that can be used:
```

Good catch on these docs, we don't need to document the connector role, it should be handled internally at the moment.
